lib/identity/config.rb

```diff
@@ -14,6 +14,10 @@ def dashboard_url
       ENV["DASHBOARD_URL"] || raise("missing=DASHBOARD_URL")
     end
 
+    def sso_base_url
+      ENV["SSO_BASE_URL"] || raise("missing=SSO_BASE_URL")
```

Comment:
Won't it throw an error anyway when they run the tests if I don't require it here?